ws-nextjs-app/pages/[service]/live/[id]/Post/styles.ts

```diff
@@ -112,4 +112,23 @@ export default {
         paddingRight: 0,
       },
     }),
+  audioPost: ({ mq, spacings }: Theme) =>
+    css({
+      '& [data-e2e="media-player"]': {
```

Comment:
I like this - targetting the SMP div by an attribute we have control of. 

Is it OK that it's a `data-e2e` attribute, or should it be `id` (which is not in our control, so could change in future...?)

Reply:
Good to know this, thanks for digging around! I've opted for a hardcoded classname - https://github.com/bbc/simorgh/pull/12031/commits/312c7954783f963c3aca0656b4d527728c1ef068 - is that OK?